src/components/KebabMenu.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react'
+import { HeaderButtons, OverflowMenu } from 'react-navigation-header-buttons'
+
+import labels from '../constants/labels.json'
+
+interface KebabMenuProps {
+  children: ReactElement
+  icon: ReactElement
```

Comment:
(optional) As I assume our kebab menu always has the three icons, we maybe can just add an optional boolean param like whiteIcon? or something?

Reply:
i found out a better thing, you can just use "currentColor" and use the color attribute for styling, so we only need one icon not white and primary
```js
<svg width="7" height="29" viewBox="0 0 7 29" xmlns="http://www.w3.org/2000/svg">
    <g fill="currentColor">
            <circle cx="3.5" cy="3.5" r="3.5"/>
            <circle cx="3.5" cy="14.5" r="3.5"/>
            <circle cx="3.5" cy="25.5" r="3.5"/>
    </g>
</svg>

Reply:
in general yes you could also use `fill="currentColor"` in the svg to style in with the css attrib `color: primary` f.e. but we use different sizes and padding f.e. in HeaderWithMenu, so i would like to keep it as an ReactElement but refactor the svg to avoid several svg only differing in color